mod_test/models.py

```diff
@@ -256,10 +256,7 @@ def get_customized_regressiontests(self) -> Any:
         rtype: list
         """
         customized_test = self.customized_tests
-        if len(customized_test) != 0:
-            regression_ids = [r.regression_id for r in customized_test]
-        else:
-            regression_ids = [r.id for r in RegressionTest.query.all()]
+        regression_ids = [r.regression_id for r in customized_test]
```

Comment:
How are we sure that the size of `customized_test` will not be zero?

Reply:
Tests are run only on 2 conditions - upon req. from GH or when admin runs customized test.
1. When the GH workflows are complete `queue_test` function is responsible to add customized test functions
2. Customized test form doesn't allow creating an empty test.

This way, for those tests whose GH Actions are not completed yet, their customized tests would be an empty list, hence we don't have to run their tests right now and wait for it to either have tests or it would get automatically canceled.

Reply:
Got it, so in the new flow, it is never possible to have an empty customized_test list.

Reply:
Yes, and we utilising this fact to wait for GH Actions, so that those tests whose GH Actions are pending are not run by the cron job...